rero_mef/authorities/marctojson/do_bnf_auth_person.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of RERO MEF.
+# Copyright (C) 2018 RERO.
+#
+# RERO MEF is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# RERO MEF is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with RERO MEF; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Marctojsons transformer skeleton."""
+
+
+# ---------------------------- Modules ----------------------------------------
+# import of standard modules
+
+import re
+
+from rero_mef.authorities.marctojson.helper import \
+    build_string_list_from_fields
+
+# third party modules
+
+# local modules
+
+
+__author__ = "Gianni Pante <Gianni Pante@rero.ch>"
+__version__ = "0.0.1"
+__copyright__ = "Copyright (c) 2009-2018 Rero, Gianni Pante"
+__license__ = "Internal Use Only"
+
+
+# ----------------------------------- Classes ---------------------------------
+
+# MrcIterator ----
+class Transformation(object):
```

Comment:
1. BnfTransformation instead? If yes do the same for the others.
2. Create a base class for the common methods.